dev-haskell/x11-xft/x11-xft-0.3.4.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.7.2.1.9999
+#hackport: flags: -pedantic
+CABAL_PN="X11-xft"
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour"
+inherit haskell-cabal
+
+DESCRIPTION="Bindings to the Xft and some Xrender parts"
+HOMEPAGE="https://hackage.haskell.org/package/X11-xft"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=">=dev-haskell/utf8-string-0.1:=[profile?]
+	>=dev-haskell/x11-1.2.1:=[profile?]
+	>=dev-lang/ghc-8.4.3:=
+	x11-libs/libXft
+"
+DEPEND="${RDEPEND}
+	>=dev-haskell/cabal-2.2.0.1
+	virtual/pkgconfig
```

Comment:
BDEPEND for pkgconfig (arguably cabal too but let's not go there for now)

Reply:
Thanks - should be sorted now.